src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -27,21 +27,17 @@ namespace ripple {
 
 namespace BuildInfo {
 
-char const* getRawVersionString ()
-{
-    static char const* const rawText =
+//------------------------------------------------------------------------------
+char const* const versionString =
```

Comment:
I recommend:
````
constexpr char const* const versionString =
````
This gives `versionString` internal linkage (file scope).  After experimenting with online compilers, I believe this incantation will satisfy all 3 of our compilers with no warnings.